tools/apiview/parsers/rust-api-parser/get-api-view/expected-outputs/docs.rs

```diff
@@ -0,0 +1,42 @@
+
+pub mod docs{
```

Comment:
At some point you'll also want to add some attributes e.g.,

```rust
#[derive(Debug)]
pub struct foo;
```

We should see most attributes (there's relatively few we shouldn't, and I don't even know if they'd show in in JSON so let's deal with them as we find them); though, remember that for `derive` macros you'll have to piece them back together. I'm not aware of any others currently.